.classpath

```diff
@@ -2,31 +2,45 @@
 <classpath>
 	<classpathentry excluding="processing/app/tools/format/|processing/app/tools/format/src/|processing/app/Trace.java|processing/app/RunnerClassLoader.java" kind="src" path="app/src"/>
 	<classpathentry kind="src" path="app/test"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
-	<classpathentry kind="lib" path="app/lib/apple.jar"/>
-	<classpathentry kind="lib" path="app/lib/ecj.jar"/>
-	<classpathentry kind="lib" path="app/test-lib/junit-4.11.jar"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
```

Comment:
This line bothers me a bit... Is it necessary to specify JavaSE-1.8 ?

Reply:
Ideally, I would like to specify Java 1.8 32 bits, but this does not seem to be possible. Not specifying a java version leads to using the user-defined workspace default, which can differ from what is used to build against by Ant and the Continuous Integration software. Since the Ant config contains `jvmRequired="1.8"`, I think it is a good idea to stick to that version. Note that later JDK versions can suffice for earlier version settings in Eclipse (e.g. JDK 1.8 can be used to build a project that's configured to build against 1.6).
I'd like to hear your thoughts / proposal on this.